### PR TITLE
Feature: Phan Plugin

### PR DESCRIPTION
**Pull request by caouecs**
## Contribution type

New plugin

## Description of change

I did a plugin for [Phan](https://github.com/phan/phan), it launches a command to find PHP files, and after it launches Phan on this list.

### CODE REVIEW

File: src/Plugin/Phan.php [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+        $warnings = 0;
+
+        foreach ($json as $data) {
+            $this->build->reportError(
+                $this->builder,
+                'phan',
+                $data['check_name']."\n\n".$data['description'],
+                $this->severity($data['severity']),
+                $data['location']['path'] ?? '??',
```
Comment (corpsee) [inline src/Plugin/Phan.php]: It doesn't work in PHP 5.6.

**Comment by corpsee** [comment]
@caouecs Excellent work! Can you fix code to support PHP 5.6 please?

**Comment by caouecs** [comment]
Done

**Comment by corpsee** [comment]
@caouecs Thanks!

**Pull request merged**